hack/verify-test-code.sh

```diff
@@ -46,6 +46,15 @@ do
     fi
 done
 
+errors_expect_no_equal=()
```

Comment:
ref:https://github.com/kubernetes/kubernetes/issues/86206 
 We need to discuss this change，right?
/cc @neolit123